src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js

```diff
@@ -1,51 +1,48 @@
-define(['angular', 'application/interviewer/tdprInterviewerModule'], function(angular, tdprInterviewerModule) {
-  tdprInterviewerModule.controller("tdprInterviewerHomeController", function($scope, tdprSlotsService, $filter, $stateParams,$timeout) {
-    $scope.slotTimes = [];
-        tdprSlotsService.getSlotsTimes().then(function(response) {
-            for(var i = 0; i < response.data.length; i++) {
-                var startTime = response.data[i].startTime.slice(0,5);
-                var endTime = response.data[i].endTime.slice(0,5);
+define(['angular', 'application/interviewer/tdprInterviewerModule'], function (angular, tdprInterviewerModule) {
+    tdprInterviewerModule.controller("tdprInterviewerHomeController", function ($scope, tdprSlotsService, $filter, $stateParams, $timeout) {
+        $scope.slotTimes = [];
+        tdprSlotsService.getSlotsTimes().then(function (response) {
+            for (var i = 0; i < response.data.length; i++) {
+                var startTime = response.data[i].startTime.slice(0, 5);
+                var endTime = response.data[i].endTime.slice(0, 5);
                 $scope.slotTimes.push(startTime + "-" + endTime);
             }
-    });
+        });
 
         $scope.startDate;
         $scope.endDate;
-
+        $scope.currentType = '1'; // 1 - available, 2 - full, 3 - init, 4 - unavailable, 5 - maybe
```

Comment:
We should make this into an enum - an angular constant object which we could inject - like tdprAvailabilityTypes
{available: 'available', full: 'full', init: 'init', unavailable: 'unavailable', maybe: 'maybe'} - do we need to use these numbers?